File: APIs/doctor_app/Alarm/setAlarm.php
<?php

include '../conn.php';

$patientID = $_POST['PATIENT_ID'];
$title = $_POST['TITLE'];
$body = $_POST['BODY'];
$dateTime = $_POST['DATE_TIME'];

$res = $conn->query("INSERT INTO `alarm` (`PATIENT_ID`,`TITLE`, `BODY`, `DATE_TIME`, `STATUS`)
VALUES ('".$patientID."','".$title."', '".$body."','".$dateTime."', 'W');");

echo json_encode($res);

?>

File: Client/dro_doctor_app/lib/APIs/AlarmAPI/SetAlarm.dart
import 'dart:convert';

import 'package:dro_doctor_app/APIs/mainURL/mainURL.dart';
import 'package:http/http.dart' as http;

class SetAlarmAPI{

  Future<bool> setAlarm({String patientID,String title,String body})async{
    String url = '${mainURL}Alarm/setAlarm.php';
    try{
      http.Response response = await http.post(url,body: {
        'PATIENT_ID' : patientID,
        'TITLE' : title,
        'BODY' : body,
        'DATE_TIME' : '${DateTime.now()}'
      });
      if(response.statusCode == 200){
        return jsonDecode(response.body);
      }else return false;
    }catch (e){
      print('cancel notification API ======)>)>)>)>)> $e');
      return false;
    }

  }

}